test_shallow_backup.py

```diff
@@ -0,0 +1,7 @@
+import pytest
+
+def test_backup_prompt():
```

Comment:
This doesn't test any functionality of the program, only that `" "` isn't in a list that's hardcoded and does not contain the string `" "`

This test can never fail... and is therefore useless.

Write a useful test and I'll approve this PR.

One example could be testing the `copying` functionalities with a variety of paths, including edge cases, like a path with escape characters in it.